### PR TITLE
ISPN-14213-re Add a statement why we provide no performance numbers

### PR DESCRIPTION
**Pull request by dfitzmau**
Jira: https://issues.redhat.com/browse/JDG-5418

Original PR: https://github.com/infinispan/infinispan/pull/10380
needs backport to 14.0.x

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @dvagnero .

I updated the PR by creating a docs-specific attribute for the title you recently updated. Perhaps we could create attributes for all the documents and house them in docs-specific attributes file?

File: documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc [hunk 1]
```diff
@@ -131,6 +131,11 @@
 :jgroups_schema: http://www.jgroups.org/schema/jgroups-5.2.xsd
 :jgroups_extras: https://github.com/jgroups-extras
 
+//
+// Document name
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc:135]:
```suggestion
// Infinispan titles
```

File: documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc [hunk 2]
```diff
@@ -131,6 +131,11 @@
...
+//
+:PerformanceSize_docname: {brandname} Performance and Sizing guide
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc:137]: That's the product title. You need:
```suggestion
:sizing_title: {brandname} performance considerations and tuning guidelines
```

File: documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 = Performance metric considerations
 {brandname} includes so many configurable combinations that determining a single formula for performance metrics that covers all use cases is not possible.  
 
-The purpose of the _{brandname} performance considerations and tuning guidelines_ document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
+The purpose of the _{PerformanceSize_docname}_ document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc:5]:
```suggestion
The purpose of the _{sizing_title}_ is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```
Think about both docs sets:
"Infinispan performance considerations and tuning guidelines" and  "Deployment Planning and Sizing Guide document"

File: documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc (hunk 2 above)
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc:137]: PerformanceSize_docname seems too long

**Comment by dvagnero** [comment]
The same attribute has to be added to the product repo

**Comment by dfitzmau** [comment]
> The same attribute has to be added to the product repo

OK, but how do I do this? What exact file do I need to update?

**Comment by dvagnero** [comment]
> > The same attribute has to be added to the product repo
> 
> OK, but how do I do this? What exact file do I need to update?

You go to gitlab and update the rhdg-attributes.adoc.

**Comment by dfitzmau** [comment]
> > > The same attribute has to be added to the product repo
> > 
> > 
> > OK, but how do I do this? What exact file do I need to update?
> 
> You go to gitlab and update the rhdg-attributes.adoc.

My PR targets GitHub. Do I need to create a separate MR for GitLab?

File: documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc (hunk 3 above)
Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc:5]: So the titles names are different from upstream and downstream? So I need ifdef statements?

**Comment by dvagnero** [comment]
> > > The same attribute has to be added to the product repo
> > 
> > 
> > OK, but how do I do this? What exact file do I need to update?
> 
> You go to gitlab and update the rhdg-attributes.adoc.

Yes you need a separate PR to change this file

File: documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc (hunk 2 above)
Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc]: Updated

File: documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc (hunk 1 above)
Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc]: UPdated

**Review by dvagnero — APPROVED** [review]
lgtm. Thanks

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**